src/main_app/sqlalchemy_db/services/settings_service.py

```
@@ -67,40 +82,64 @@ def update_setting(
     with get_session() as session:
-        job = SettingRecord(key=key, title=title, value=value, value_type=value_type)
-        session.add(job)
-        session.commit()
-        session.refresh(job)
-        return job
+        setting = SettingRecord(key=key, title=title, value=value, value_type=value_type)
+        session.add(setting)
+        try:
+            session.commit()
+            return True
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)